packages/nimble-components/src/rich-text-editor/styles.ts

```diff
@@ -1,12 +1,163 @@
 import { css } from '@microsoft/fast-element';
 import { display } from '@microsoft/fast-foundation';
-import { bodyFont, bodyFontColor } from '../theme-provider/design-tokens';
+import {
+    bodyFont,
+    bodyFontColor,
+    borderHoverColor,
+    borderRgbPartialColor,
+    borderWidth,
+    smallDelay,
+    standardPadding
+} from '../theme-provider/design-tokens';
 
 export const styles = css`
-    ${display('flex')}
+    ${display('inline-flex')}
 
     :host {
         font: ${bodyFont};
+        outline: none;
         color: ${bodyFontColor};
+        width: auto;
+        height: 100%;
+        flex-direction: column;
+        position: relative;
+        --ni-private-hover-indicator-width: calc(${borderWidth} + 1px);
+        --ni-private-footer-section-height: 40px;
+        ${
+            /** Minimum width is added to accommodate all the possible buttons in the toolbar and to support the mobile width. */ ''
+        }
+        min-width: 360px;
+    }
+
+    .container {
+        display: flex;
+        flex-direction: column;
+        height: inherit;
```

Comment:
Generally speaking "inherit" for height is probably not what you want, particularly for sizing like width and height.

See the discussions in the following thread and linked codepen examples from the textarea control which should be a relevant model for the rich-text-editor: https://github.com/ni/nimble/pull/457#discussion_r834596232